tutorials/supres_e2e.py

```diff
@@ -0,0 +1,174 @@
+import io
+import numpy as np
+import onnx
+import onnxruntime
+import os
+import torch
+import torch.nn as nn
+import torch.utils.model_zoo as model_zoo
+
+# ref: https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html
+
+
+_this_dirpath = os.path.dirname(os.path.abspath(__file__))
+_data_dirpath = os.path.join(_this_dirpath, 'data')
+_onnx_model_filepath = os.path.join(_data_dirpath, 'supres.onnx')
```

Comment:
It looks like there are two 'supres.onnx' are same? Any idea of deduplication?

Reply:
Yes. It's an oversight. I will remove the one in tutorial sub folder.

Reply:
Done.